Jack/Script.fsx

```diff
@@ -78,6 +78,22 @@ Property.check <| forAll {
 }
 
 //
+// Conditional Properties
+//
+
+fun (x, y) -> (x > 0 && y > 0) ==> (x * y > 0)
+|> Property.forAll (Gen.zip Gen.int Gen.int)
+|> Property.check
```

Comment:
I think you have discovered the limits of 32-bit integers 😃 

```
2147483647 = 2^31 - 1 (max 32-bit number)
2147484021 = 879 * 2443099
```

Only `374` higher than the actual maximum 32-bit number, so the shrinking works well 👍